crm_rma_prodlot_invoice/README.md

```diff
@@ -0,0 +1,18 @@
+CRM Claim Prodlot Invoice
+=========================
+
+When invoice is created from sale order, the move_id field
+filled with information in picking and procurement.
+
+It mainly contains the following features:
+------------------------------------------
+
+* Filled move_id in lines of invoice when sale order is validated and invoice is created before picking
+* Filled move_id in lines of invoice when sale order is validated and picking is created before invoice
```

Comment:
good sorry.

can you make a gqq on this two lines too split them to 80 cols. please (better readability in plain text.